doc/how-to/how-to-setup-aws-s3-submissions.md

```diff
@@ -0,0 +1,26 @@
+# How to setup AWS S3 submissions
+
+## Context
+
+When building a submission service, the "Send" component offers an option "Upload to AWS S3" which uploads the application data payload to PlanX's private S3 Bucket - which then in turn allows a council to receive a notification via a Power Automate Webhook and run a script to securely retrieve the application JSON file, download any associated user-uploaded files, generate custom documents, and save the application to their local Microsoft SharePoint environment.
+
+There's full documentation on how this method works in the PlanX Service Specification on Notion here.
+
+Once a council has confirmed they're cleared from their IT dept to use this method (eg allowed to receive external webhook requests) & shared a Power Automate Webhook URL, there's a few steps we need to take:
+
+1. Add the council-provied URL to `team_integrations.power_automate_webhook_url` as plain text via the production Hasura console
```

Comment:
```suggestion
1. Add the council-provided URL to `team_integrations.power_automate_webhook_url` as plain text via the production Hasura console
```

Reply:
Actually I did just google 'provied' and apparently it is a word so sorry if that was actually what you intended here!


> definition.com.co
> http://www.definition.com.co › provied
> 1. [ adjective ] established beyond doubt. Synonyms. proven. Examples. "a proven liar" "a Soviet leader of proven shrewdness" ; *. Similar Spelling. proved ...
>

Reply:
An actual typo - thanks for catching !